### PR TITLE
Attempt to get on-foot detection working, so EDDN data sends

### PR DESCRIPTION
**Pull request by Athanasius**
This introduces a new monitor state variable `on_foot`.

Currently this will only be true when you login at a station.  There's only `Disembark` when you get off a shuttle at one, with no new `Location`/`Docked` event to fill in the necessary data to check against CAPI data.

### CODE REVIEW

**Comment by Athanasius** [comment]
This should be ready to merge now, although could probably use some non-Alpha testing to ensure I didn't introduce regressions.

**Comment by Athanasius** [comment]
I'll s/on_foot/OnFoot/g in this tomorrow, given the rest of monitor.state is CamelCase.

File: monitor.py [hunk 1]
```diff
@@ -67,6 +67,7 @@ class EDLogs(FileSystemEventHandler):  # type: ignore # See below
     _RE_CANONICALISE = re.compile(r'\$(.+)_name;')
     _RE_CATEGORY = re.compile(r'\$MICRORESOURCE_CATEGORY_(.+);')
     _RE_LOGFILE = re.compile(r'^Journal(Alpha|Beta)?\.[0-9]{12}\.[0-9]{2}\.log$')
+    _RE_SHIP_ONFOOT = re.compile(r'^(FlightSuit|UtilitySuit_Class.)$')
```
Comment (A-UNDERSCORE-D) [inline monitor.py:70]: Is it confirmed whether or not `UtilitySuit_Class` is non-decimal? if its always a decimal this should be \d

Comment (Athanasius) [reply monitor.py:70]: It's `"name": "UtilitySuit_Class1"` for the Maverick suit, but I didn't want to assume it'd just be digits.

What we need is a new Journal doc specifying what those can be, or to be far enough into the Alpha to try out other suits.  Do NOT assume Frontier are going to be consistent on such.  Even 'single character' might turn out to not be enough.

So it'll do for now.  Change it as we learn more.

Comment (A-UNDERSCORE-D) [reply monitor.py:70]: sounds good

**Review by A-UNDERSCORE-D — APPROVED** [review]

**Pull request merged**